src/tickets.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Tickets
+---
+
+<a href="https://pretix.eu/pyconuk/PyConUK2022" target="_blank">Buy your PyCon UK 2022 tickets now</a>
+
+<div class="box box_yellow">
+  <h3>Please note</h3>
+  <p>As part of our return we are temporarily only offering tickets that cover the full conference. We are hoping to return to usual in 2023!</p>
+  <p>There are no specific childrens' tickets this year. However, ticket holders may bring two under-16s to the conference.</p>
```

Comment:
up to two under-18s

Reply:
Updated.

Kinda amused at the thought of "all ticket holders must be accompanied by precisely two under-18s" now...